Add fzf-marks integration support

tleoutline opened the pull request:
Add integration with [fzf-marks](https://github.com/urbainvaes/fzf-marks) with 3 options:
```
@sessionx-fzf-marks-mode 'off'
@sessionx-fzf-marks-file '~/.fzf-marks'
@sessionx-bind-fzf-marks 'ctrl-g'
```

Currently needed to specify the fzf-marks file manually if not in the default location (`~/.fzf-marks`).

omerxx commented:
Hey @tleoutline 
I think this is a rather niche plugin that's easily solved with zoxide and it's alternatives. 
That said, if this is required by many users I'm happy to consider merging it, could you please also fix the conflicts and add a readme explanation?

tleoutline commented:
I agree that it's a niche feature, and most use cases might overlap with zoxide, though I think that being able to manually set a location list for jumping is quite nice imo. That said, I've fixed the conflicts and added a section to the readme. Feel free to pull if you see fit.
